presets/TestSamples/4.3/ctzsnzooze_5in_4S_Race.txt

```diff
@@ -0,0 +1,140 @@
+# Title: 5" 4S racer ctzsznooze
+# FirmwareVersion: 4.3
+# Category: MIXED
+# Official: true
+# Keywords: 5in, 4S, race
+# Author: Chris Thompson ctzsnooze
+# Description: Basic 5" 4S race setup for 4.3 by ctzsnooze
+# Description: assumes a decent radio link
+
+# General settings
+
+set level_race_mode = OFF
+set mixer_type = LEGACY
+set gyro_cal_on_first_arm = OFF
+set deadband = 0
+set yaw_deadband = 0
+set runaway_takeoff_prevention = OFF
```

Comment:
Silently removing a critical safety feature is not acceptable.

Reply:
Good point.  Some race tracks have very sudden turns after takeoff and it can be difficult to ensure that runaway prevention doesn't trip when it shouldn't, so i have it off on all my race setups.
I'll include a comment.

Reply:
I didn't mean add a comment and leave it in. I meant to not disable a safety feature. A comment about how it can be disabled in the unlikely event of a problem is ok. But that comment should not encourage disabling or exaggerate the risk. The feature has been enabled for millions of flights and pretty much for every race lap flown without problem.

Reply:
OK... in that case, what values would you recommend so for a racer that turns hard nearly immediately after takeoff, so that the runaway protection disables it self really quickly and reliably?

Reply:
I'm with @etracer65 on this. We don't must remove any safety feature with presets. The user must disable it conscious and knowing what he is doing.

Reply:
> OK... in that case, what values would you recommend so for a racer that turns hard nearly immediately after takeoff, so that the runaway protection disables it self really quickly and reliably?

I don't think any settings changes are needed. It's actually a fallacy that a hard move immediately after takeoff will trigger runaway takeoff prevention. The feature works based on the pidsum growing very large for a longish period of time. The pidsum must exceed 600 (60.0 as displayed in blackbox) and stay above that limit for 75ms. A quad must be poorly tuned for this to be inadvertently triggered. I would assume that the entire point of a "racer" preset is that it contains settings to make the pid controller track setpoint well and thus not have a concern with large and persistent error. Further the code contains logic to further accelerate deactivation under high throttle conditions as you would expect early after a race launch.